web/template.py

```diff
@@ -895,7 +895,13 @@ def _compile(self, code):
 
     def __call__(self, *a, **kw):
         __hidetraceback__ = True
-        return self.t(*a, **kw)
+        try:
+            return self.t(*a, **kw)
+        except TypeError as e:
+            print(
+                "To fix this error, please see https://webpy.org/docs/0.3/templetor#errors"
+            )
+            raise e
```

Comment:
I see couple of issues with this:
- print is going to stdout instead of stderr. 
- putting links in the code and error messages is quite risky as the links become stale and we wouldn't even know. Please keep a note to fix this link when we release 1.0.

@iredmail

Reply:
We can add a test case to ensure the URL is still valid.